05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
Thanks. I still don't think it's reasonable to insist I add a minutes field when no one else seems to have to bother.